src/dodal/devices/i24/aperture.py

```diff
@@ -0,0 +1,35 @@
+from enum import Enum
+
+from ophyd_async.core import StandardReadable
+from ophyd_async.epics.motion import Motor
+from ophyd_async.epics.signal import epics_signal_rw
+
+
+class ApPosition(str, Enum):
+    IN = "In"
+    OUT = "Out"
+    ROBOT = "Robot"
+    MANUAL = "Manual Mounting"
+
+
+class AperturePositioner(StandardReadable):
+    def __init__(self, prefix: str, name: str = "") -> None:
+        self.pos_select = epics_signal_rw(ApPosition, prefix + "MP:SELECT")
+        super().__init__(name)
```

Comment:
Should: Why is this in a new class? Can we not just have the signal in the `Aperture` class?